CMakeLists.txt

```diff
@@ -0,0 +1,92 @@
+cmake_minimum_required(VERSION 3.15)
+
+set(AVR_MCU attiny44 CACHE STRING "MCU")
+set(AVR_L_FUSE 0xE2 CACHE STRING "Low Fuse")
+set(AVR_H_FUSE 0xDD CACHE STRING "High Fuse")
+set(AVR_E_FUSE 0xFE CACHE STRING "Extended Fuse")
+set(AVRDUDE_PROGRAMMER avrispmkII CACHE STRING "Programmer")
```

Comment:
Edit readme to describe what to change in order to target a different AVR programmer.